part-3/pk-sim-compounds-definition-and-work-flow.md

```diff
@@ -280,7 +276,7 @@ Sometimes enzymes that catalyze a metabolic degradation process can also bind th
 
 ### Metabolism
 
-Depending on the available experimental information you can either define process types in the **Metabolizing Enzymes** branch or the **Total Hepatic Clearance** branch. Please note that the calculation sheet offered for metabolizing enzymes refers to the liver in case of intrinsic clearance processes and in all other cases to the organ in which the respective enzyme is expressed. Using this calculation sheet, input values will be transferred to specific clearance values which are then used in the simulation. The sheet is only meant to help the user with the calculations. However, processes defined here may also be applied to other organs given that relevant expression levels are appropriately defined in the individual.
+Depending on the available experimental information you can either define process types in as **Metabolizing Enzymes** or as **Total Hepatic Clearance**. Please note that the calculations offered for metabolizing enzymes refer to the liver in case of intrinsic clearance processes and in all other cases to the organ in which the respective enzyme is expressed. Using this calculation sheet, input values will be transferred to specific clearance values which are then used in the simulation. The sheet is only meant to help the user with the calculations. However, processes defined here may also be applied to other organs given that relevant expression levels are appropriately defined in the individual.
```

Comment:
I have hard time understanding this... Calculation sheet? The generall flow is very weird.

Reply:
😄